recipes/holoviews-1.5.0/meta.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+  name: holoviews
+  version: 1.5.0
+
+source:
+  path: .
```

Comment:
The current PyPI URL is rather horrible:

https://pypi.python.org/packages/96/d2/588631da98d437ee6432999efb6e7f72e626e00f32d28fe154f016f425ec/holoviews-1.5.0.tar.gz

I am hoping this is because the package has just been uploaded and that a better URL will be made available soon.